src/main/java/com/google/maps/DirectionsApiRequest.java

```diff
@@ -15,207 +15,18 @@
 
 package com.google.maps;
 
-import static com.google.maps.internal.StringJoin.join;
-
-import com.google.maps.DirectionsApi.RouteRestriction;
 import com.google.maps.model.DirectionsRoute;
-import com.google.maps.model.LatLng;
-import com.google.maps.model.TrafficModel;
-import com.google.maps.model.TransitMode;
-import com.google.maps.model.TransitRoutingPreference;
-import com.google.maps.model.TravelMode;
-import com.google.maps.model.Unit;
-
-import org.joda.time.ReadableInstant;
 
 /**
  * Request for the Directions API.
+ *
+ * <p>This request generates an elided result with just the routes.</p>
```

Comment:
Added commentary to both DirectionsApiRequest.java and FullDirectionsApiRequest.java